replication-api/src/main/java/com/connexta/replication/api/data/SiteKind.java

```diff
@@ -11,9 +11,22 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package com.connexta.ion.replication.api;
+package com.connexta.replication.api.data;
 
-public enum NodeAdapterType {
-  DDF,
-  ION
+/** The different kind of sites that are currently supported. */
+public enum SiteKind {
```

Comment:
❓ Why do we need this? seems odd that we have a SiteType and a SiteKind.

Reply:
Because it might change the way we replicate if we know the site is a tactical one vs a regional one (e.g. pipe smaller, prioritizing metadata over resource, ...)